ods_ci/tasks/Resources/Provisioning/Hive/provision.robot

```diff
@@ -151,7 +151,7 @@ Wait For Cluster To Be Ready
     Run Keyword And Ignore Error    Watch Hive Install Log    ${pool_namespace}    ${install_log_file}
     Log    Verifying that Cluster '${cluster_name}' has been provisioned and is running according to Hive Pool namespace '${pool_namespace}'      console=True    # robocop: disable:line-too-long
     ${provision_status} =    Run Process
-    ...    oc -n ${pool_namespace} wait --for\=condition\=Provisioned\=True cd ${pool_namespace} --timeout\=10m
+    ...    oc -n ${pool_namespace} wait --for\=condition\=ProvisionFailed\=False cd ${pool_namespace} --timeout\=15m
```

Comment:
Me and @kobihk found that it could take a very long time until Hive cluster updates the "Provisioned" condition to True, so another condition, which exists in both Hive versions should be checked. Note that due to API changes between the versions, not all condition types exist or have similar status after cluster is provisioned, but "ProvisionFailed" exists on both.

Reply:
Okay, it doesn't make much sense to me why Provisioned state takes more time to set and ProvisionFailed not, but I have no actual objections if it will work for us :slightly_smiling_face:

Reply:
I also was skeptical but when I ran with this PR the deployment passed as expected
Thank you Noam!!